openlibrary/solr/update_work.py

```diff
@@ -1139,6 +1141,37 @@ def toxml(self):
         if self.keys:
             return make_delete_query(self.keys)
 
+
+def solr8_update(
+        reqs: List[Union[str, UpdateRequest, DeleteRequest]],
+        commit_within=60_000,
+        skip_id_check=False,
+        solr_base_url: str = None,
+) -> None:
+    """This will replace solr_update once we're fully on Solr 8.7+"""
+    req_strs = (r if type(r) == str else r.toxml() for r in reqs if r)  # type: ignore
```

Comment:
```suggestion
    req_strs = (r if type(r) is str else r.toxml() for r in reqs if r)  # type: ignore
```

pep8

Reply:
Going to punt on this one; I don't believe it's necessary in this case? @dhruvmanila @cclauss Do you by any chance know?

Reply:
As described in PEP8, do not directly compare types but use `isinstance(r, str)` instead.